site/content/contribute/server/system_console.md

```diff
@@ -2,14 +2,17 @@
 title: "System Console"
 heading: "The Mattermost System Console"
 description: "Find out how to add fields, expose settings, and make settings available for non-admins in the System Console."
-date: 2019-10-09T13:38:26-04:00
+date: 2022-03-15T13:38:26-04:00
 weight: 5
 ---
 
 ## Adding fields to the configuration
 
 In order to add fields to the configuration, you need to modify `model/config.go` in the server by adding the desired field to one of the structs such as `ServiceSettings` and setting its default value in the corresponding `SetDefaults` method.
 
+Note that some of the configuration values are collected as telemetries. The telemetry definitions are defined in the `services/telemetry` package. Once a configuration is added, it should be added to telemetry package. If the configuration value is not going to be collected as a telemetry, a `// telemetry: none` comment must be added to prevent the [configtelemetry](https://github.com/mattermost/mattermost-govet#included-analyzers) check from failing.
```

Comment:
```suggestion
Note that some of the configuration values are collected as telemetries. The telemetry definitions are defined in the `services/telemetry` package. Once a configuration is added, it should be added to the telemetry package. If the configuration value is not going to be collected as a telemetry, a `// telemetry: none` comment must be added to prevent the [configtelemetry](https://github.com/mattermost/mattermost-govet#included-analyzers) check from failing.
```